### PR TITLE
Wristwatches

### PR DESCRIPTION
**Pull request by DeityLink**
Inspired by #11712 of all things
image

image
_Also the hand tips glow in the dark._

imageimage

image

:cl:
* rscadd: Added wristwatches to the game, a new uniform accessory. Their hands move in real time on their sprite, and you can click it to get the current station time down to the second. Can be purchased from the merch computer at the Cargo Lobby.
* rscadd: They can also extremely rarely be found from maintenance loot spawners, and tourists coming out of Ancient Cryopods have one equipped as a reference to the old long-time layering glitch that caused their right arm to appear to wear a red strap.
* rscadd: Captains spawn with a unique golden wristwatch. Although there is a small chance of getting another one from the Vault's safe.
* rscadd: There is also a sleek black variant that can be rarely found from maint spawners. Nuclear Operatives spawn equipped with those, as time is of the essence in their field.
* rscadd: Finally a pocket watch variant can be dug up frome house/dwellings digsites on the asteroid. Let your local xenoarchaeologist know if you're interested in getting one.
* imageadd: Special thanks to terrel for providing the gold, black, and pocket watch sprites.

### CODE REVIEW

File: code/modules/maps/spawners/spawners.dm [hunk 1]
```diff
@@ -1255,7 +1256,7 @@
 	/obj/item/clothing/suit/space/rig/soviet,
 	/obj/item/clothing/suit/space/rig/nazi,
 	/obj/abstract/spawn_all/dredd_gear,
-	/obj/item/clothing/glasses/emitter
+	/obj/item/clothing/glasses/emitter,
```
Comment (DeityLink) [inline code/modules/maps/spawners/spawners.dm:1261]:
```suggestion
	/obj/item/clothing/glasses/emitter
```

**Comment by Durgen** [comment]
the pull request that saved /vg/station

**Comment by SnakeEater14** [comment]
I really really want this

please be available somewhere ez so I can rush it every roundstart

**Comment by SonixApache** [comment]
add a digital one as an archeology find

**Comment by Eneocho** [comment]
> 
> 
> add a digital one as an archeology find

It would make more sense for the analog one to be the archeology find, because old. Though it'd be cool to see either way.

**Comment by DeityLink** [comment]
> I really really want this
> 
> please be available somewhere ez so I can rush it every roundstart

I'd argue that the merch computer is easy enough to rush and 50$ isn't much for a mechanical watch.

**Comment by Eneocho** [comment]
The sprite doesn't have a seconds hand though. How do you tell the time down to seconds without a seconds hand.

**Comment by DeityLink** [comment]
There is a seconds hand. It's just too thin to appear on the sprite :)

**Comment by DeityLink** [comment]
image

More watches, courtesy of boxed.

**Comment by SonixApache** [comment]
👌

**Comment by Kurfursten** [comment]
Icon conflicts

**Comment by DeityLink** [comment]
Fixed conflicts

**Pull request merged**